[2024-03-07T05:39:04.746+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: Extraçao_de_dados_do_linkedin.Extraçao_dos_dados manual__2024-03-07T05:34:26.491150+00:00 [queued]>
[2024-03-07T05:39:04.758+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: Extraçao_de_dados_do_linkedin.Extraçao_dos_dados manual__2024-03-07T05:34:26.491150+00:00 [queued]>
[2024-03-07T05:39:04.759+0000] {taskinstance.py:2170} INFO - Starting attempt 2 of 4
[2024-03-07T05:39:04.815+0000] {taskinstance.py:2191} INFO - Executing <Task(PythonOperator): Extraçao_dos_dados> on 2024-03-07 05:34:26.491150+00:00
[2024-03-07T05:39:04.824+0000] {standard_task_runner.py:60} INFO - Started process 7174 to run task
[2024-03-07T05:39:04.826+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'Extraçao_de_dados_do_linkedin', 'Extraçao_dos_dados', 'manual__2024-03-07T05:34:26.491150+00:00', '--job-id', '113', '--raw', '--subdir', 'DAGS_FOLDER/dag_exemplo.py', '--cfg-path', '/tmp/tmppmifn1m7']
[2024-03-07T05:39:04.828+0000] {standard_task_runner.py:88} INFO - Job 113: Subtask Extraçao_dos_dados
[2024-03-07T05:39:04.918+0000] {task_command.py:423} INFO - Running <TaskInstance: Extraçao_de_dados_do_linkedin.Extraçao_dos_dados manual__2024-03-07T05:34:26.491150+00:00 [running]> on host 4fca1e22de4a
[2024-03-07T05:39:05.105+0000] {taskinstance.py:2480} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='Extraçao_de_dados_do_linkedin' AIRFLOW_CTX_TASK_ID='Extraçao_dos_dados' AIRFLOW_CTX_EXECUTION_DATE='2024-03-07T05:34:26.491150+00:00' AIRFLOW_CTX_TRY_NUMBER='2' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-03-07T05:34:26.491150+00:00'
[2024-03-07T05:39:15.950+0000] {warnings.py:109} WARNING - /opt/***/dags/scrap/main.py:50: GuessedAtParserWarning: No parser was explicitly specified, so I'm using the best available HTML parser for this system ("lxml"). This usually isn't a problem, but if you run this code on another system, or in a different virtual environment, it may use a different parser and behave differently.

The code that caused this warning is on line 50 of the file /opt/***/dags/scrap/main.py. To get rid of this warning, pass the additional argument 'features="lxml"' to the BeautifulSoup constructor.

  soup = BeautifulSoup(data)

[2024-03-07T05:39:27.582+0000] {logging_mixin.py:188} INFO - ok
[2024-03-07T05:39:31.597+0000] {logging_mixin.py:188} INFO - 60
[2024-03-07T05:39:35.664+0000] {logging_mixin.py:188} INFO - Atualmente na página: 0
[2024-03-07T05:39:46.043+0000] {logging_mixin.py:188} INFO - Loader automático...
[2024-03-07T05:39:52.057+0000] {logging_mixin.py:188} INFO - 60
[2024-03-07T05:39:55.848+0000] {logging_mixin.py:188} INFO - Atualmente na página: 1
[2024-03-07T05:40:06.198+0000] {logging_mixin.py:188} INFO - Loader automático...
[2024-03-07T05:40:12.217+0000] {logging_mixin.py:188} INFO - 60
[2024-03-07T05:40:15.945+0000] {logging_mixin.py:188} INFO - Atualmente na página: 2
[2024-03-07T05:40:26.305+0000] {logging_mixin.py:188} INFO - Loader automático...
[2024-03-07T05:40:33.210+0000] {logging_mixin.py:188} INFO - ['NA']
[2024-03-07T05:40:38.014+0000] {logging_mixin.py:188} INFO - ['NA']
[2024-03-07T05:40:43.423+0000] {logging_mixin.py:188} INFO - ['Nível de experiência', 'Não aplicável', 'Tipo de emprego', 'Tempo integral', 'Função', 'Gerenciamento e Manufatura', 'Setores', 'Fabricação de produtos alimentícios e bebidas']
[2024-03-07T05:40:48.264+0000] {logging_mixin.py:188} INFO - ['NA']
[2024-03-07T05:40:53.096+0000] {logging_mixin.py:188} INFO - ['NA']
[2024-03-07T05:40:58.462+0000] {logging_mixin.py:188} INFO - ['Nível de experiência', 'Pleno-sênior', 'Tipo de emprego', 'Tempo integral', 'Função', 'Administração, Consultoria e Estratégia/Planejamento', 'Setores', 'Tecnologia, Informação e Internet']
[2024-03-07T05:41:03.298+0000] {logging_mixin.py:188} INFO - ['NA']
[2024-03-07T05:41:08.118+0000] {logging_mixin.py:188} INFO - ['NA']
[2024-03-07T05:41:13.542+0000] {logging_mixin.py:188} INFO - ['Nível de experiência', 'Não aplicável', 'Tipo de emprego', 'Tempo integral', 'Função', 'Educação e Treinamento', 'Setores', 'Assessoria em relações públicas e comunicação']
[2024-03-07T05:41:18.346+0000] {logging_mixin.py:188} INFO - ['NA']
[2024-03-07T05:41:23.143+0000] {logging_mixin.py:188} INFO - ['NA']
[2024-03-07T05:41:28.844+0000] {logging_mixin.py:188} INFO - ['Nível de experiência', 'Não aplicável', 'Tipo de emprego', 'Tempo integral', 'Função', 'Pesquisa, Analista e Tecnologia da informação', 'Setores', 'Assessoria em relações públicas e comunicação']
[2024-03-07T05:41:34.335+0000] {logging_mixin.py:188} INFO - ['Nível de experiência', 'Não aplicável', 'Tipo de emprego', 'Tempo integral', 'Função', 'Tecnologia da informação', 'Setores', 'Comércio varejista']
[2024-03-07T05:41:39.153+0000] {logging_mixin.py:188} INFO - ['NA']
[2024-03-07T05:41:44.544+0000] {logging_mixin.py:188} INFO - ['Nível de experiência', 'Não aplicável', 'Tipo de emprego', 'Tempo integral', 'Função', 'Desenvolvimento de negócios e Vendas', 'Setores', 'Fabricação de máquinas e equipamentos de uso industrial específico']
[2024-03-07T05:41:49.424+0000] {logging_mixin.py:188} INFO - ['NA']
[2024-03-07T05:41:54.306+0000] {logging_mixin.py:188} INFO - ['NA']
[2024-03-07T05:41:59.106+0000] {logging_mixin.py:188} INFO - ['NA']
[2024-03-07T05:42:04.531+0000] {logging_mixin.py:188} INFO - ['Nível de experiência', 'Não aplicável', 'Tipo de emprego', 'Tempo integral', 'Função', 'Gerenciamento e Manufatura', 'Setores', 'Transporte, armazenagem e correio']
[2024-03-07T05:42:08.876+0000] {local_task_job_runner.py:302} WARNING - State of this instance has been externally set to failed. Terminating instance.
[2024-03-07T05:42:08.880+0000] {process_utils.py:131} INFO - Sending 15 to group 7174. PIDs of all processes in the group: [7174]
[2024-03-07T05:42:08.881+0000] {process_utils.py:86} INFO - Sending the signal 15 to group 7174
[2024-03-07T05:42:08.882+0000] {taskinstance.py:2450} ERROR - Received SIGTERM. Terminating subprocesses.
[2024-03-07T05:42:10.854+0000] {python.py:201} INFO - Done. Returned value was: None
[2024-03-07T05:42:10.869+0000] {taskinstance.py:1138} INFO - Marking task as SUCCESS. dag_id=Extraçao_de_dados_do_linkedin, task_id=Extraçao_dos_dados, execution_date=20240307T053426, start_date=20240307T053904, end_date=20240307T054210
[2024-03-07T05:42:11.048+0000] {process_utils.py:79} INFO - Process psutil.Process(pid=7174, status='terminated', exitcode=0, started='05:39:04') (7174) terminated with exit code 0
